README.md

```diff
@@ -2729,6 +2729,9 @@ State of the VLAN. Valid values are 'active', 'suspend', and keyword 'default'.
 Whether or not the vlan is shutdown. Valid values are 'true', 'false' and
 keyword 'default'.
 
+##### `fabric_control`
```

Comment:
Please add a caveat for this property to the caveats table to indicate it is only supported on `n7k` and that the module minimum version is `1.3.0`

https://github.com/rkorlepa/cisco-network-puppet-module/blob/feature/vlan_fabric_control/README.md#caveats-9